arch/riscv/Kconfig

```diff
@@ -128,6 +128,46 @@ config GEN_IRQ_VECTOR_TABLE
 config NUM_IRQS
 	int
 
+config HAS_MONITOR_HART
+	bool "Does SOC have monitor hart with different capabilities to application harts"
+	default n
```

Comment:
Booleans default to `n`. This is redundant

Reply:
done